src/Serilog.Extensions.Hosting/Serilog.Extensions.Hosting.csproj

```diff
@@ -13,21 +13,39 @@
     <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
     <PackageId>Serilog.Extensions.Hosting</PackageId>
     <PackageTags>serilog;aspnet;aspnetcore;hosting</PackageTags>
+    <PackageIcon>serilog-extension-nuget.png</PackageIcon>
     <PackageIconUrl>http://serilog.net/images/serilog-extension-nuget.png</PackageIconUrl>
     <PackageProjectUrl>https://github.com/serilog/serilog-extensions-hosting</PackageProjectUrl>
-    <PackageLicenseUrl>http://www.apache.org/licenses/LICENSE-2.0</PackageLicenseUrl>
+    <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
     <RepositoryUrl>https://github.com/serilog/serilog-extensions-hosting</RepositoryUrl>
     <RepositoryType>git</RepositoryType>
     <GenerateAssemblyVersionAttribute>false</GenerateAssemblyVersionAttribute>
     <RootNamespace>Serilog</RootNamespace>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Serilog" Version="2.8.0" />
-    <PackageReference Include="Serilog.Extensions.Logging" Version="3.0.1" />
-    <PackageReference Include="Microsoft.Extensions.Hosting.Abstractions" Version="2.1.0" />
-    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="2.1.0" />
-    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="2.1.0" />
+    <None Include="images/serilog-extension-nuget.png" Pack="true" PackagePath="/" />
+  </ItemGroup>
+
+  <PropertyGroup>
+    <LatestVersion>[*, 9999.0)</LatestVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Serilog" Version="$(LatestVersion)" />
+    <PackageReference Include="Serilog.Extensions.Logging" Version="$(LatestVersion)" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
+    <PackageReference Include="Microsoft.Extensions.Hosting.Abstractions" Version="2.1.*" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="2.1.*" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="2.1.*" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.1'">
+    <PackageReference Include="Microsoft.Extensions.Hosting.Abstractions" Version="3.1.*" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="3.1.*" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="3.1.*" />
```

Comment:
All these packages are targeted to `netstandard2.0`. No need to multitargeting `  <TargetFrameworks>netstandard2.0;netcoreapp3.1</TargetFrameworks>.`

Reply:
Ok, that's what was all about my remarks in the comment ;)
That the solution i would prefer too

but that also means that people using `AspNetCore 2.x` won't resolved this assembly anymore and need to pin it to the current version.

Can you validate that you are ok with it ?

Reply:
> but that also means that people using AspNetCore 2.x won't resolved this assembly anymore and need to pin it to the current version.

I don't understand.

Reply:
This is the Metapackage used for AspNetCore 2.x :
https://www.nuget.org/packages/Microsoft.AspNetCore.App

Open the dependency list, it's built by dependending on `Microsoft.Extension.xxxxx` 2.x
So if this nuget stays `netstandard2.0` (As it should, and as you suggest too), and now depends on `Microsoft.Extension.xxxxx` 3.x, then not sure what's going to happen.

Again the solution for user still using `AspNetCore 2.x` would just be to pin this dependency, that's all

Reply:
That is, in essence, we are talking about two incompatible versions of the package for the same `netstandard2.0` platform? And they are incompatible because of their dependencies, which changed the major version. Right?

Then this is really some problem. But you should keep in mind that nothing prevents using the new version of this package with 3.1 dependencies in applications that do not use AspNetCore at all. New 3.1 libs are just upgraded libs but still target to `netstandard2.0`

I would say that AspNetCore 2.x users were initially limited by the ability to evolve into new versions of libraries due to restrictions introduced by the authors of this metapackage.

Reply:
> That is, in essence, we are talking about two incompatible versions of the package for the same netstandard2.0 platform? And they are incompatible because of their dependencies, which changed the major version. Right?

This is basically a Major version change yes, so in order to be ... up to date, people needs to update :D
I always find it funny to say but that basically it :
* people using up to date aspnetcore would just get this lib up to date
* people not updating their own app would just pin this lib, that's all

Reply:
That is, in any case, we should not retarget the package to netcoreapp3.1. Right?

Reply:
If the decision is 
* to use "SemVer" / "just update" and "consumer needing OLD version of Microsoft (i mean still using 2.x)" => then yes
* To let both aspnetcore 2.x and 3.x consumer => there was never a solution since packaged existed ... so the targetframework here is an opportunity that would only fix it for NetCoreApp3.1 (AspNetCore 3.1 / UWP / WinForm / Wpf / Console using Netcoreapp3.1).

Remember that .Net 5 is goign to be a big game changer for that ;)
As we speak Mono is being merge in `dotnet/runtime` ;)

Reply:
again i insist on the fact that consumer that keeps using `aspnetcore 2.x` are willing NOT TO UPDATE, meaning they already pinned their own system.

So yes i'm in favor of target `netstandard2.0`, and move to `PackageReference Microsoft.Extensions.xxxx 3.x`
But that's not my decision to take

Reply:
Let's listen to @nblumhardt opinion.